code_challenges/advent_of_code/repetition_code.py

```diff
@@ -0,0 +1,50 @@
+def get_messages():
+    with open("./code_challenges/advent_of_code/message_input") as message_input:
+        messages = message_input.read().split("\n")
+    return messages
+
+
+def break_up_messages():
+    messages = get_messages()
+    character_breakdown = {}
```

Comment:
You never use this dictionary as a dictionary, why not a list of character count dicts?